### ISSUE TITLE
Add placeholder for date and time in session boundary

### ISSUE THREAD

**Issue by tajemniktv** [mrbuilder1961/WheresMyChatHistory#198]
Hi, 
I was wondering if something like this is possible: 

**Describe the solution you'd like**
Placeholder similiar to $worldname, except for date and/or time in session boundary category.

**Comment by mrbuilder1961** [comment]
yeah this definitely could be possible! what time would the timestamp reflect, like when the world was loaded or something different?
